app/assets/javascripts/fae/form/_ajax.js

```diff
@@ -121,7 +121,33 @@ Fae.form.ajax = {
 
         // remotipart returns data inside textarea, let's grab it from there
         if ($html.is('textarea')) {
-          $html = $( $html.val() );
+          if ( $html.length > 1 ) {
+            var textarea;
+
+            var isTextarea = function(items) {
```

Comment:
textarea.text() is not a jQuery object which we still need for the rest of the function, $.parseHTML will return a jQuery object.

I didn't want to remove any previously existing functionality because it seemed like I was dealing with a strange case, so I kept the check for .is() in there with the response it got originally in the else of the nested if/else block. The first .is('textarea') will determine if there is any textarea present, then we can recursively check for the exact textarea; if there is no textarea, skip it. We recursively check because as you pointed out, while unlikely, perhaps the textarea is nested another level deep. I think it's overkill, but I was addressing your concern above.